packages/fiori/src/themes/SearchItem.css

```diff
@@ -55,4 +59,30 @@
 	width: 1rem;
 	height: 1rem;
 	box-sizing: content-box;
+}
+
+:host([description]) {
+	height: auto;
+	min-height: var(--_ui5_list_item_base_height);
+}
+
+:host([description]) .ui5-search-item-content {
+	padding-top: var(--_ui5_search_item_content_top_bottom_padding);
+	padding-bottom: var(--_ui5_search_item_content_top_bottom_padding);
+}
+
+:host([description]) .ui5-search-item-subheading {
+	display: inline-block;
+}
+
+.ui5-search-item-subheading {
```

Comment:
why not description? isn't it confusing to have a text property and render it as a .ui5-search-item-heading and a description and name it .ui5-search-item-subheading. Both of them are not heading semantically, why naming them like that.